### ISSUE TITLE
Does synthetic_examples need any saved models?

### ISSUE THREAD

**Issue by GinGinWang** [Shai128/mqr#1]
Hi, 
I am trying to run the synthetic examples  and there is error said that 
_No such file or directory: 'models_save/nonlinear_cond_banana_k_dim_1/seed=0/mode=CVAE_loss=KL_z_dim=3_kl=0.01/cvae'_
I have created the corresponding directory while it still does not work. It seems that the synthetic examples need some pretrained models.   Should I run any experiments before the synthetic_examples?

Thanks a lot for your help.

**Comment by Shai128** [comment]
Hi,
Thanks for your comment!
To run the synthetic example you need the pre-trained CVAE model saved in that directory. For that purpose, run `train_vae.py` with the relevant parameters (mode=CVAE, loss=KL, z_dim=3, kl=0.01), and the script will save the trained model to the directory.

Good luck!
